frontend/src/routes/+error.svelte

```diff
@@ -0,0 +1,107 @@
+<script>
+	import { Button } from '@hyvor/design/components';
+	import { page } from '$app/state';
+	import { Footer, FooterLinkList, Header } from '@hyvor/design/marketing';
+	import IconBoxArrowUpRight from '@hyvor/icons/IconBoxArrowUpRight';
+	import IconGithub from '@hyvor/icons/IconGithub';
+</script>
+
+<Header logo="https://hyvor.com/api/public/logo/core.svg" subName="Relay" darkToggle={false}>
```

Comment:
don't duplicate. Use the current header and footer. If it's in +layout.svelte, abstract it first